tlsr9/CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ zephyr_include_directories(common)
 zephyr_include_directories(drivers/${SOC}/compatibility_pack)
 zephyr_include_directories(drivers/${SOC}/reg_include)
 zephyr_include_directories(drivers/${SOC})
+if (CONFIG_SOC_RISCV_TELINK_B92)
+zephyr_include_directories(drivers/${SOC}/lib/include/pke)
```

Comment:
@TelinkKrystian 
eccp_curve.h, pke_utility.h, pke.h added to 
zephyrproject/modules/hal/telink/tlsr9/drivers/B92
no need to duplicate

Reply:
Done